mcu_program/common_src/lms7002m_filters.c

```diff
@@ -170,12 +170,14 @@ uint8_t TuneRxFilterSetup(const float_type rx_lpf_IF)
 */
 #endif
     Modify_SPI_Reg_bits(G_TIA_RFE, g_tia_rfe);
-    if(g_pga_rbb == 31)
+    if(g_pga_rbb == 31) {
         Modify_SPI_Reg_bits(G_PGA_RBB, 22);
-    else
+    }
+    else {
         Modify_SPI_Reg_bits(G_PGA_RBB, g_pga_rbb);
+    }
 
-	{
+    {
```

Comment:
I am certain that it was. It was just bad indentation. The brackets are there to limit scope of variable to help compiler optimize memory usage. You can see this a lot in the code used by MCU as it has very limited resources.